tensorflow/lite/micro/recording_micro_allocator.cc

```diff
@@ -163,10 +163,12 @@ TfLiteStatus RecordingMicroAllocator::AllocateNodeAndRegistrations(
 
   TfLiteStatus status =
       MicroAllocator::AllocateNodeAndRegistrations(model, subgraph_allocations);
+
+  RecordAllocationUsage(allocations,
+                        recorded_node_and_registration_array_data_);
```

Comment:
This was called in a subgraph loop, so it would count the same allocations `num_subgraphs` times. The same holds for line 198 below.